gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -943,6 +943,10 @@ gst_tensor_filter_install_properties (GObjectClass * gobject_class)
           "input and output of the tensor filter. "
           "With this option, the output caps is always in the format of flexible tensors.",
           FALSE, G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
+  g_object_class_install_property (gobject_class, PROP_INVOKE_ASYNC,
+      g_param_spec_boolean ("invoke-async", "Enable asynchronous invoke",
+          "Use this when the sub-plugin receives an input and generates multiple outputs asynchronously.",
+          FALSE, G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
```

Comment:
You set
```
if (!prop->invoke_async) {
    throw std::runtime_error (
        "llamacpp must invoke output asynchronously. Set `invoke_async=true`");
  }
```
Then, I think it would be more appropriate to set invoke_async to true by default.

Reply:
In most cases where tensor_filter is used, invoke-async should be false. It seems better to set it to true only in special cases like llama.cpp.
Thank you.